de/nmichael/efa/gui/EfaCloudConfigDialog.java

```diff
@@ -0,0 +1,293 @@
+import de.nmichael.efa.gui.util.EfaMenuButton;
+import de.nmichael.efa.util.International;
+import de.nmichael.efa.util.LogString;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.util.Vector;
+
+/**
+ * A dialog to trigger efaCloud synchronisation activities.
+ */
+public class EfaCloudConfigDialog extends BaseTabbedDialog implements IItemListener {
+
+    private static final String ACTIVATION_INFO = "ACTIVATION_INFO";
+    private static final String BUTTON_EFACLOUD_ACTIVATE = "BUTTON_EFACLOUD_ACTIVATE";
+    private static final String BUTTON_EFACLOUD_DEACTIVATE = "BUTTON_EFACLOUD_DEACTIVATE";
+    private static final String BUTTON_EFACLOUD_START = "BUTTON_EFACLOUD_START";
+    private static final String BUTTON_EFACLOUD_SYNCH_UPLOAD = "BUTTON_EFACLOUD_SYNCH_UPLOAD";
+    private static final String BUTTON_EFACLOUD_PAUSE = "BUTTON_EFACLOUD_PAUSE";
+    private static final String BUTTON_EFACLOUD_DELETE = "BUTTON_EFACLOUD_DELETE";
+
+    private AdminRecord admin;
+    private final JDialog parent;
+    private final TxRequestQueue txq;
+
+    public EfaCloudConfigDialog(JDialog parent, AdminRecord admin) {
+        super(parent, International.getStringWithMnemonic("EfaCloud"), International.getStringWithMnemonic("Schließen"),
+                null, true);
+        this.parent = parent;
+        this.txq = TxRequestQueue.getInstance();
+        iniItems(admin);
+    }
+
+    /**
+     * Forward the key action to the superclass BaseDialog for handling
+     *
+     * @param evt action which occurred.
+     */
+    public void keyAction(ActionEvent evt) {
+        _keyAction(evt);
+    }
+
+    /**
+     * Initialize all items on theis dialog.
+     *
+     * @param admin the admin which provides the rights to use. It must have the "isAllowedAdministerProjectLogbook()"
+     *              authorisation.
+     */
+    private void iniItems(AdminRecord admin) {
+        this.admin = admin;
+        Vector<IItemType> guiItems = new Vector<IItemType>();
+        String category;
+        IItemType item;
+        int projectStorageType = (Daten.project == null) ? IDataAccess.TYPE_FILE_XML : Daten.project
+                .getProjectStorageType();
+        boolean isEfaRemote = projectStorageType == IDataAccess.TYPE_EFA_REMOTE;
+        boolean isXML = projectStorageType == IDataAccess.TYPE_FILE_XML;
+        boolean isEfaCloud = projectStorageType == IDataAccess.TYPE_EFA_CLOUD;
+
+        String queueState = (txq == null) ? "UNDEFINED" : TxRequestQueue.QUEUE_STATE.get(txq.getState());
+        String advice = (isEfaRemote) ? International
+                .getString("Für efaRemote kann der efaCloud-Modus nicht aktiviert werden.") : (isXML) ?
+                International.getString("Aktuelles Projekt auf efaCloud umstellen.") + "\n" : International
+                .getMessage("Efacloud Server Kommunikation steuern.\nAktueller Status: {state}", queueState) + "\n";
```

Comment:
Internationalisierte Strings duerfen keine \n enthalten. Bitte brich den String in zwei separate getString/getMessage calls.
Achte bitte auch auf einheitliche Schreibweise von "efaCloud". Ich schreibe efa immer - auch am Satzanfang - mit kleinem "e". efaCloud sollte also immer "efaCloud" geschrieben werden.